code/drasil-docLang/lib/Drasil/DocumentLanguage/TraceabilityMatrix.hs

```diff
@@ -79,6 +79,7 @@ tableShows r end = refS r +:+ S "shows the" +:+ plural dependency `S.ofThe` (end
 layoutUIDs :: [TraceViewCat] -> ChunkDB -> [UID] -> [UID]
 layoutUIDs a c e = concatMap (filter (`elem` (Map.keys $ c ^. traceTable)) . (\ x -> x e c)) a
 
+
```

Comment:
however this extra blank line snuck in, it should be taken back out

Reply:
Ok I will remove that right now!